### DISCUSSION TITLE
Old About - How to fix it?

### DISCUSSION THREAD

**Discussion by DanielSnor** [mastodon/mastodon#31113] [General]
Hello. I am struggling with About. I have the @zpravobot.news server (currently 4.2.9), which still has the old About information and Rules on the homepage (resp. on the About page) from the time I ran that server on 3.5.8. The mascot picture was also not shown. 

The server was upgraded from 3.5.8 to 4.2.1 (I had to wait for Debian 12) and then to 4.2.9.

Could someone help me with solving this? Your assistance would be greatly appreciated. Thank you.

![Screenshot 2024-07-23 at 14 29 03](https://github.com/user-attachments/assets/bbc5ce69-1659-4478-b6fd-878637a4c3e2)

**Comment by mjankowski** [comment]
I see this page is now populated - https://zpravobot.news/about - looks like you figured this out?
